src/test/scala/nl.knaw.dans.easy.deposit/docs/DatasetMetadataSpec.scala

```diff
@@ -162,4 +163,113 @@ class DatasetMetadataSpec extends TestSupportFixture {
+    }
+  }
+
+  "DatasetMetadata.spatialBoxes" should "only report spatial points of the box that are invalid" in {
+    val boxes: String =
+      """{
+        |	"spatialBoxes": [{
+        |		"north-west": 2,
+        |		"south-east": 3,
+        |		"north": 4,
+        |		"east": 5,
+        |		"south": 9,
+        |		"west": 10
+        |	}],
+        |}""".stripMargin
+    DatasetMetadata(boxes) should matchPattern {
+      case Failure(ide: InvalidDocumentException) if ide.getMessage == "invalid DatasetMetadata: don't recognize {\"spatialBoxes\":{\"north-west\":2,\"south-east\":3}}" =>
+    }
+  }
+
+  "DatasetMetadata.spatialCoverages" should "only report someCoverage if is supplied instead of spatialCoverages" in {
+    val alteredData = createCorruptMetadataJsonString(" \"spatialCoverages\"", " \"someCoverage\"")
+    DatasetMetadata(alteredData) should matchPattern {
+      case Failure(ide: InvalidDocumentException) if ide.getMessage == "invalid DatasetMetadata: don't recognize {\"someCoverage\":[{\"scheme\":\"dcterms:ISO3166\",\"value\":\"string\",\"key\":\"string\"}]}" =>
+    }
+  }
+
+  "DatasetMetadata.Dates" should "only report the dates that are not correct" in {
+    val dates =
+      """{
+        |	"dates": [{
+        |			"scheme": "dcterms:W3CDTF",
+        |			"value": "2018-05-31",
+        |			"qualifier": "dcterms:created"
+        |		},
+        |		{
+        |			"invalidOne": "invalid",
+        |			"invalidValue": "2018-05-31",
+        |			"invalidQualifier": "dcterms:created"
+        |		}
+        |	]
+        |}""".stripMargin
+    DatasetMetadata(dates) should matchPattern {
+      case Failure(ide: InvalidDocumentException) if ide.getMessage == """invalid DatasetMetadata: don't recognize {"dates":{"invalidOne":"invalid","invalidValue":"2018-05-31","invalidQualifier":"dcterms:created"}}""" =>
+    }
+  }
+
+  "DatasetMetadata.creators" should "only report the part of the creators that are wrong" in {
+    val creatorValid = """{"titles": "Msc", "initials": "H.A.M.", "surname": "Boter", "ids": [ { "scheme": "DAI", "value":  "93313935x"}, {"scheme": "BSN", "value": "1234"} ], "organization" :"DANS"}"""
+    val creatorInvalidIdElement = """{"titles": "Msc", "initials": "B.A.M.", "surname": "Hoter", "ids": [ { "scheme": "DAI", "value":  "93313935Z"}, {"scheme": "BSN", "value": "1235", "organization": "overheid"} ], "organization" :"DANS"}"""
+    val creatorInvalidElementAtRootLevel = """{"titles": "Aartshertog", "FirstName": "jan-willem-hendrik", "surname": "Oranje", "ids": [ { "scheme": "DAI", "value":  "93313935y"}, {"scheme": "BSN", "value": "9999"} ], "organization" :"DANS"}"""
+    //TODO moet hij hier niet struikelen over het veld role[0].waarde?
```

Comment:
Try whether it shows this error when the others are fixed.

Rather use multi-line input than variables.